WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -22,13 +22,49 @@ class WordPress_Sniffs_WP_I18nSniff extends WordPress_Sniff {
 
 	/**
 	 * These Regexes copied from http://php.net/manual/en/function.sprintf.php#93552
+	 * and adjusted for better precision and updated specs.
 	 */
-	const SPRINTF_PLACEHOLDER_REGEX = '/(?:(?<!%)(%(?:[0-9]+\$)?[+-]?(?:[ 0]|\'.)?-?[0-9]*(?:\.[0-9]+)?[bcdeEufFgGosxX]))/';
+	const SPRINTF_PLACEHOLDER_REGEX = '/(?:
```

Comment:
Oh and just to be clear - the first two changes in the regex is what allows the new unit test / test case by @ocean90 to pass correctly.